config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  # Hikes routes
+
+  get "/hikes" => "hikes#index"
+  get "/hikes/:id" => "hikes#show"
```

Comment:
you can use Rails' `resources` syntax here:

`resources :hikes, only: %i[index show]`

will give you the same ReSTful routes you have here.

Reply:
routes have been updated to reflect this